lib/phlex/rails/context.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+ApplicationControllerForPhlex = Class.new(ActionController::Base)
+
+module Phlex
+  module Rails
+    module Context
+      def render(partial = nil, **kwargs)
+        if @_rendering || @_rendering_block
+          _raw ApplicationControllerForPhlex.new.render_to_string(partial:, locals: kwargs)
```

Comment:
This trick about capturing the first positional argument as the partial name and passing the rest of the `kwargs` as locals seems risky, don't you think?

After some more digging inside the source code of Rails, maybe we could use this:

```suggestion
          _raw ApplicationControllerForPhlex.new.view_context.render(...)
```

That way we don't have to override the signature of the `render` method that is used inside Rails views. What do you think?

Reply:
Oh I like this idea.